kyuubi-common/src/test/scala/org/apache/kyuubi/operation/JDBCTestHelper.scala

```diff
@@ -97,4 +122,10 @@ trait JDBCTestHelper extends KyuubiFunSuite {
   def withJdbcStatement(tableNames: String*)(f: Statement => Unit): Unit = {
     withMultipleConnectionJdbcStatement(tableNames: _*)(f)
   }
+
+  def withJdbcPrepareStatement(
```

Comment:
where does it be used ?

Reply:
it has been used for test, to cover 'DEALLOCATE PREPARE' syntax support